src/icewatch/render_facilities_map.py

```diff
@@ -160,7 +193,7 @@ def render_html(
             <div class="legend-row"><span class="legend-icon" style="background:rgba(244,67,54,0.7);width:30px;height:30px;border-radius:50%;border:2px solid #222;"></span><span class="legend-label">500+ people</span></div>
         </div>
     </main>
-    <div id="last-updated">last updated {formatted_date}</div>
+    <div id="last-updated">last checked {formatted_date}</div>
```

Comment:
can we show both this last checked field and the last updated field? i.e. so we can see the last time we checked if there's updates and the last time we actually updated the dashboard